modules/custom/az_publication/config/install/views.view.az_publications_curate.yml

```diff
@@ -846,7 +846,8 @@ display:
           required: true
       display_description: ''
       display_extenders: {  }
-      path: admin/az-publication/curation/%node
```

Comment:
This change is made because the new proposed role should have access to curate publications, but does not have access to `Use the administration pages` permission. This is different from the permission `view the administration theme`, which the role does have.